src/Dflydev/FigCookies/SetCookie.php

```diff
@@ -227,38 +229,37 @@ public function __toString() : string
         return implode('; ', $cookieStringParts);
     }
 
-    public static function create(string $name, ?string $value = null) : self
+    public static function create(string $name, ?string $value = null): self
     {
-        return new static($name, $value);
+        return new self($name, $value);
     }
 
-    public static function createRememberedForever(string $name, ?string $value = null) : self
+    public static function createRememberedForever(string $name, ?string $value = null): self
     {
         return static::create($name, $value)->rememberForever();
     }
 
-    public static function createExpired(string $name) : self
+    public static function createExpired(string $name): self
     {
         return static::create($name)->expire();
     }
 
-    public static function fromSetCookieString(string $string) : self
+    public static function fromSetCookieString(string $string): self
     {
         $rawAttributes = StringUtil::splitOnAttributeDelimiter($string);
 
         $rawAttribute = array_shift($rawAttributes);
 
         if (! is_string($rawAttribute)) {
-            throw new \InvalidArgumentException(sprintf(
+            throw new InvalidArgumentException(sprintf(
                 'The provided cookie string "%s" must have at least one attribute',
                 $string
             ));
         }
 
-        list ($cookieName, $cookieValue) = StringUtil::splitCookiePair($rawAttribute);
+        [$cookieName, $cookieValue] = StringUtil::splitCookiePair($rawAttribute);
 
-        /** @var SetCookie $setCookie */
-        $setCookie = new static($cookieName);
+        $setCookie = new self($cookieName);
```

Comment:
class is not final, means this is a behavior change

Reply:
Reverted.

Reply:
this one isn't reverted yet

Reply:
Indeed, can this one get updated, too?